src/app/pages/partners/partners.component.scss

```
@@ -0,0 +1,130 @@
+.partners-content a:not(.header-anchor) {
+  @apply underline decoration-dotted;
+}
+
+.partners-content a:not(.header-anchor):hover {
+  @apply text-gray-500;
+}
+
+.partners-paragraph {
+  @apply my-10 text-lg text-white;
+  text-align: justify;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)